app/views/Dashboard/index.tsx

```diff
@@ -135,6 +136,12 @@ function Dashboard() {
         COUNTRIES_AND_OUTBREAKS,
     );
 
+    useEffect(() => {
+        if (activeTab === 'country' && !filterValues) {
+            setFilterValues({ country: 'AFG' });
+        }
+    }, [activeTab, filterValues]);
+
```

Comment:
Let's avoid using useEffect.

This will only set country to be "AFG" on the second render

Reply:
Cannot be avoided.